pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -11972,6 +11972,19 @@ final: prev: {
     meta.hydraPlatforms = [ ];
   };
 
+  nvim-treesitter-legacy = buildVimPlugin {
+    pname = "nvim-treesitter-legacy";
+    version = "0.10.0-unstable-2025-05-24";
+    src = fetchFromGitHub {
+      owner = "nvim-treesitter";
+      repo = "nvim-treesitter";
+      rev = "42fc28ba918343ebfd5565147a42a26580579482";
+      hash = "sha256-CVs9FTdg3oKtRjz2YqwkMr0W5qYLGfVyxyhE3qnGYbI=";
+    };
+    meta.homepage = "https://github.com/nvim-treesitter/nvim-treesitter/";
+    meta.hydraPlatforms = [ ];
+  };
+
```

Comment:
This shouldn't be in generated, it should be a non-generated temporary implementation. Generated is for plugins that will be updated via the updater, this branch is unsupported/unmaintained upstream.